int64 key_i
int64 key_j
float32[3] xyz
float32[3] rpy
float32[6] sigmas
